client/src/api/auth.js

```diff
@@ -0,0 +1,14 @@
+export const signApi = (email, password) => {
+  const opts = {
```

Comment:
Various code style guides prefer full words rather than abbreviations, so we could rename `opts` to `options`